packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -355,14 +381,12 @@ export default class Calendar extends React.Component {
       return
     }
 
-    let classes = ['react-datepicker-navigation', 'react-datepicker-navigation--next']
-
-    if (this.props.todayButton) {
-      classes.push('react-datepicker-navigation--next--with-today-button')
-    }
-
-    return <a
-      className={cx(classes)}
+    return <Button
+      className={cx('react-datepicker-navigation--next')}
+      icon={<span className={cx('next-month-icon')} />}
+      isIconOnly
+      variant="utility"
+      text="Next Month"
```

Comment:
Todo: Get translation

Reply:
Updated in this commit: https://github.com/cerner/terra-framework/pull/866/commits/6ea023ec6abb090fc1b5d30e23041f9ddbddad13